### DISCUSSION TITLE
docs.google.com - Is it possible to exclude canvas elements from being inverted?

### DISCUSSION THREAD

**Discussion by rlas** [darkreader/darkreader#9897] [Q&A] [answered]
I found how to exclude the entire canvas from the dark theme; simply had to uncomment the ".doc-editor canvas" line in dev tools.
This is exactly what I desired for "docs.google.com/documents/".
It's also nearly perfect for "docs.google.com/spreadsheets" except for the column and row headers.

Is it possible to only keep the cells area of the spreadsheet unchanged?
That is, to have Dark Reader apply to everything but the value cells.

I've looked around with the chrome dev tool and can't figure out how to accomplish this.  I have pretty much no experience with anything beyond basic HTML and CSS so I'm hoping someone more experienced than me can either point me in the right direction or tell me if it's even possible in the first place.

**Comment by Gusted** [comment, marked as answer]
Please note that changes in the devtools aren't permanent and could be overwritten with Dark Reader updates. You want a [dedicated extension](https://github.com/openstyles/stylus) to load custom CSS into pages, as for this scenario you want:

```css
#doc-editor canvas {
    filter: none !important;
}
```

But as for the spreadsheets, the cells, columns and row headers are one element(a `<canvas>` element), so it's not possible to keep them inverted and leave the cells alone.

**Reply by General1Chaos** [reply]
For future reference, either you have a typo or google changed the ID of the relevant element in the last year, because it needs to be `#docs-editor` rather than `#doc-editor` in order to work now. 
```css
#docs-editor canvas {
    filter: none !important;
}
```
